CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### New features since last release
 
+* Added a `GenericDevice` (PennyLane device name: `qiskit.generic`) that accepts a backend
```

Comment:
slight name preference: could we call it `RemoteDevice` instead? Same with the rest of the changes.
```suggestion
* Added a `RemoteDevice` (PennyLane device name: `qiskit.remote`) that accepts a backend
```